qa/L0_java_resnet/ResnetTest.java

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights
+// reserved.
```

Comment:
Remove the `(c)` so that this fits on one line. It is failing `L0_copyrights` due to this. Please fix for all 3 files below:
```
+ qa/common/check_copyright.py --year=2023 CITATION.cff CMakeLists.txt CONTRIBUTING.md Dockerfile.QA Dockerfile.sdk Dockerfile.win10.min LICENSE NVIDIA_Deep_Learning_Container_License.pdf README.md TRITON_VERSION Triton-CCLA-v1.pdf build build.py compose.py deploy docker docs pyproject.toml qa src
copyright year is not recognized for qa/L0_java_memory_growth/MemoryGrowthTest.java: // Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights
copyright year is not recognized for qa/L0_java_resnet/ResnetTest.java: // Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights
copyright year is not recognized for qa/L0_java_sequence_batcher/SequenceTest.java: // Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights
```

Reply:
Thanks for spotting this and recommending a fix. I think I thought it was previously failing and had missed the actual error. Fixed per your recommendation. This now passes in the latest CI.